### PR TITLE
feat: support for different brands

### LINKED ISSUE #355: In production, change favicon to be OpenShift icon
For example, this one: https://assets.openshift.net/content/subdomain/favicon32x32.png

### PR DESCRIPTION
**Pull request by pmuir**
Fixes #355

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 639c2098cbe1ba6c084548853951cb4658e35a29
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: config/webpack.prod.js [hunk 1]
```diff
@@ -39,6 +41,7 @@ const PUBLIC_PATH = process.env.PUBLIC_PATH || '/';
 const BUILD_NUMBER = process.env.BUILD_NUMBER;
 const BUILD_TIMESTAMP = process.env.BUILD_TIMESTAMP;
 const BUILD_VERSION = process.env.BUILD_VERSION;
+const FABRIC8_BRANDING = process.env.FABRIC8_BRANDING || 'fabric8';
```
Comment (joshuawilson) [inline config/webpack.prod.js]: Why aren't we using the OS icon here in prod? You have it provisioned but don't use it.

File: config/branding.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+exports.assets = {
+  fabric8: {
+    favicon: {
+      path: './src/assets/icon/fabric8_icon.svg'
```
Comment (joshuawilson) [inline config/branding.js]: I think this is a nice way to handle this. :+1:

File: config/webpack.prod.js (hunk 1 above)
Comment (joshuawilson) [inline config/webpack.prod.js]: oh, now I see it in the docker file setup

File: config/branding.js (hunk 2 above)
Comment (pmuir) [inline config/branding.js]: I still really need the svg for the openshift logo, adam is hunting it down.

**Pull request merged**